src/libs/StartupTimer/index.native.ts

```diff
@@ -6,7 +6,7 @@ import type StartupTimer from './types';
  */
 const startupTimer: StartupTimer = {
     stop: () => {
-        NativeModules.StartupTimer.stop();
+        (NativeModules.StartupTimer as StartupTimer).stop();
```

Comment:
Let's [define types for all NativeModules in this PR](https://stackoverflow.com/questions/62450379/react-native-extend-nativemodules-typescript-types), what do you think? 

You can extend NativeModulesStatic in `src/types/modules/react-native.d.ts` file.

Reply:
Sounds good, I'll change it